MaxText/inference_mlperf/llama_offline_run.sh

```diff
@@ -117,7 +117,7 @@ else
   export DATASET_TYPE=full
   export DATASET_PATH=${DATA_DISK_DIR}/processed-data.pkl
   export TOTAL_SAMPLE_COUNT=24576
-  export USER_CONFIG=user.conf
+  export USER_CONFIG=user.conf # NOTE: you may need to change this to be an absolute path
```

Comment:
Nit: an example .conf file can be found here: `maxtext/MaxText/inference_mlperf/user100.conf`